.pre-commit-config.yaml

```diff
@@ -0,0 +1,58 @@
+minimum_pre_commit_version: "2.9.2"
+repos:
+  - repo: meta
+    hooks:
+      - id: identity
+      - id: check-hooks-apply
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "" # Can not be removed, so leave this empty.
+    hooks:
+      - id: prettier
+  - repo: local
+    hooks:
+      - id: no-relative-imports
+        name: No relative imports.
+        description: Use absolute imports only.
+        entry: "^\\s*from\\s+\\."
+        language: pygrep
+      - id: isort
+        name: iSort - Sorts imports.
+        description: Sorts your import for you.
+        entry: isort
+        args: ["--settings-path=pyproject.toml"]
+        language: python
+        types: [python]
+        require_serial: true
+        additional_dependencies:
+          - isort
+      - id: black
+        name: Black - Auto-formatter.
+        description: Black is the uncompromising Python code formatter. Writing to files.
+        entry: black
+        args: ["--config=pyproject.toml"]
```

Comment:
Is this needed? I thought that black found it here automatically 🤔

Reply:
Removed :) good catch!